lib/msal-browser/apiReview/msal-browser.api.md

```diff
@@ -993,6 +993,7 @@ export { INetworkModule }
 // @public
 export type InitializeApplicationRequest = {
     correlationId?: string;
+    appId?: string;
```

Comment:
appId is already a part of the PCA config, do we really need to accept this on initialize as well?

Reply:
+1. You can extract `clientId` from `auth.clientId` config attribute